app/models/profile.rb

```diff
@@ -3,19 +3,21 @@
 #
 # Table name: profiles
 #
-#  id         :bigint           not null, primary key
-#  user_id    :bigint
-#  first_name :string(255)
-#  last_name  :string(255)
-#  dob        :date
-#  address    :string(255)
-#  postcode   :string(255)
-#  city       :string(255)
-#  country    :string(255)
-#  state      :integer          default("drafted"), unsigned
-#  metadata   :text(65535)
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
+#  id           :bigint           not null, primary key
```

Comment:
Why it relates to this PR?

Reply:
We havent updated annotates for a long time (its not automatic in barong). As soon as i have added new model - i've re-runed an annotations. You can see there is no difference in schema but only in annotations in models)